webapp/src/components/modals/create_update_issue/create_update_issue.jsx

```diff
@@ -122,24 +155,25 @@ export default class CreateIssueModal extends PureComponent {
             return null;
         }
 
+        const repoName = this.state.repo.name ?? this.state.repo;
```

Comment:
Why is `this.state.repo` sometimes an object, and sometimes a string? Is it ever not defined?

Reply:
When we are selecting a repo while creating an issue then `this.state.repo` is storing an object and when we are editing an issue then the repository is fixed so in that case `this.state.repo` is storing a string.

Reply:
@Nityanand13 Can we make it so we consistently store one data type, rather than making this state variable hold two different potential data types?

Reply:
Yes, we can do that.